package.json

```diff
@@ -153,5 +156,8 @@
       "app/assets/javascripts",
       "node_modules"
     ]
+  },
+  "engines": {
+    "yarn": ">= 1.0.0"
```

Comment:
What does this do?

Reply:
This line was added by the migrate away package. It actually just specifies what our system runs that is basically what package manager we use with its version info. It will generate a warning when using with npm install as we have specified yarn as our engine. 
https://docs.npmjs.com/files/package.json#engines